frontend/src/pages/modelRegistry/screens/ModelRegistry.tsx

```diff
@@ -24,10 +24,8 @@ const ModelRegistry: React.FC<ModelRegistryProps> = ({ ...pageProps }) => {
   return (
     <ApplicationsPage
       {...pageProps}
-      title={
-        <TitleWithIcon title="Registered models" objectType={ProjectObjectType.deployedModels} />
-      }
-      description="View and manage your registered models."
+      title={<TitleWithIcon title="Model registry" objectType={ProjectObjectType.deployedModels} />}
+      description="View and manage all of your registered models. Registering models to model registry allows you to manage their content, metadata, versions, and user access settings."
```

Comment:
There was a further tweak to this text in a comment from @bredamc here, that thread wasn't resolved: https://docs.google.com/document/d/119Q_pgVphI-9HqCG3Uk6a1WJN5H8vYIr_sdosKDsXBA/edit?disco=AAABSu4BLBg

"View and manage all of your registered models. Registering a model in a model registry enables you to manage the model content, metadata, versions, and user access settings."

Should we go with this update @bredamc @yih-wang ?

Reply:
Ah, this also appears to conflict with the proposed text in this newer document - https://docs.google.com/document/d/1-YvEYVaVJXWKGGTsZTsVyxzKpaHYfSvGNzz6rIrlvDE/edit#heading=h.ebw7t5jmjuw

"Select a model registry to view and manage your registered models. Model registries provide a structured and organized way to store, share, version, deploy, and track models."

Which is correct @bredamc @yih-wang @kaedward ? Assuming the latter?

Reply:
@mturley Thanks for pointing this out! I didn't notice this conflict... So these two versions of microcopy are provided by @kaedward and @bredamc in two different docs. @kaedward @bredamc Can you take a look at the two versions mentioned above by @mturley? This is the paragraph that shows on the main page of the Model Registry as the overall description of the feature. Can you vote for a final version?

Reply:
Added two comments in the MVP Content Handoff doc.

Reply:
I've updated text based on the most recent document. Looks like there are a couple outstanding comments still, including the summary discussed here. Maybe if we can't get answers by tomorrow we merge as-is? Seems like the text will continue to change past MVP, so I'm not sure if these types of questions should block the PR @mturley